content/nextflow-on-aws-batch/60_cleanup/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Clean up the Environment"
+chapter: true
+weight: 60
+---
+
+## Clean Up!
+
+### At an AWS event
+
+In case you are using an AWS provided environment you do not need to clean up, as this environment is cleaned up for you.
+
+### On your own
+
+#### S3
+
+```bash
+for b in $(aws s3 ls |awk '/nextflow-spot-batch/{print $3}' |xargs); do echo "# aws s3 rb --force s3://$b" ; aws s3 rb --force s3://$b ;done
+```
+
+#### ECR
+
+```
+for x in $(aws ecr --region=${AWS_REGION}  describe-repositories |jq -r '.repositories[] | .repositoryName' |xargs);do echo "# aws ecr --region=${AWS_REGION} delete-repository --force --repository-name=$x" ; aws ecr --region=${AWS_REGION} delete-repository --force --repository-name=$x;done
```

Comment:
Hey, this raises a flag for me; This will clean all the repositories in my account on that region. In my case it deleted all the other projects I had in that region. We can really mess up if people run things on their own accounts. I'd suggest we filter and only list based on a regex for the project

Reply:
I made the clean up a bit more carefull and put a big warning at the top